In [7]:
import pandas as pd
import networkx as nx
from collections import Counter

# caminhos
solution_path = "../data/optimization/solution_simple.csv"

# escolha qual rede usar para pegar nome/partido
# pode ser a de covotação ou de coautoria, desde que os IDs batam com os da otimização
gexf_path = "../data/networks/covoting-2019_2020_2021_2022-20251208-051415.gexf"
# ou, se quiser usar a de coautoria:
# gexf_path = "../data/networks/coauthorship-network-2019_2020_2021_2022-20251208-051609.gexf"

# 1) ler solução
solution = pd.read_csv(solution_path)

# filtrar só os selecionados
selected_ids = solution.loc[solution["selected"] == 1, "id"].tolist()

# 2) ler a rede GEXF
G = nx.read_gexf(gexf_path)

# dependendo de como o grafo foi salvo, os nós podem ser strings
# então vamos trabalhar com strings
selected_ids_str = [str(i) for i in selected_ids]

print("Deputados selecionados:\n")

parties = []

for node_id in selected_ids_str:
    if node_id not in G.nodes:
        # se não encontrar, tenta sem cast, só por segurança
        if node_id.isdigit() and int(node_id) in G.nodes:
            attrs = G.nodes[int(node_id)]
        else:
            print(f"(AVISO) Deputado {node_id} não encontrado no grafo.")
            continue
    else:
        attrs = G.nodes[node_id]

    nome = attrs.get("label", "(sem nome)")
    partido = attrs.get("party", "(sem partido)")

    print(f"{node_id}: {nome} ({partido})")
    parties.append(partido)

# 3) contagem por partido
print("\nQuantidade de deputados por partido na coalizão:\n")

counter = Counter(parties)
for partido, qtd in sorted(counter.items()):
    print(f"{partido}: {qtd}")



Deputados selecionados:

107283: Gleisi Hoffmann (PT)
109429: Benes Leocádio (UNIÃO)
116379: Darci de Matos (PSD)
121948: Adriano do Baldy (PP)
122158: Pedro Paulo (PSD)
122974: Pedro Lucas Fernandes (UNIÃO)
132504: Enio Verri (PT)
133439: André Figueiredo (PDT)
133810: Marcelo Moraes (PL)
136811: Afonso Hamm (PP)
137070: Vicentinho Júnior (PP)
139285: Lídice da Mata (PSB)
141391: Arnaldo Jardim (CIDADANIA)
141398: Carlos Zarattini (PT)
141408: Cleber Verde (MDB)
141411: Dagoberto Nogueira (PSDB)
141421: Eduardo da Fonte (PP)
141450: Hugo Leal (PSD)
141458: João Carlos Bacelar (PL)
141459: João Maia (PP)
141464: José Airton Félix Cirilo (PT)
141470: José Guimarães (PT)
141488: Paulo Teixeira (PT)
141518: Paulinho da Força (SOLIDARIEDADE)
141531: Rodrigo de Castro (UNIÃO)
141555: Vinicius Carvalho (REPUBLICANOS)
143084: Tiago Dimas (PODE)
143632: Domingos Neto (PSD)
150418: Márcio Marinho (REPUBLICANOS)
152605: Glauber Braga (PSOL)
154178: Milton Vieira (REPUBLICANOS)
156190: Marcel van

'''
Deputados selecionados:

107283: Gleisi Hoffmann (PT)
109429: Benes Leocádio (UNIÃO)
116379: Darci de Matos (PSD)
121948: Adriano do Baldy (PP)
122158: Pedro Paulo (PSD)
122974: Pedro Lucas Fernandes (UNIÃO)
132504: Enio Verri (PT)
133439: André Figueiredo (PDT)
133810: Marcelo Moraes (PL)
136811: Afonso Hamm (PP)
137070: Vicentinho Júnior (PP)
139285: Lídice da Mata (PSB)
141391: Arnaldo Jardim (CIDADANIA)
141398: Carlos Zarattini (PT)
141408: Cleber Verde (MDB)
141411: Dagoberto Nogueira (PSDB)
141421: Eduardo da Fonte (PP)
141450: Hugo Leal (PSD)
141458: João Carlos Bacelar (PL)
141459: João Maia (PP)
141464: José Airton Félix Cirilo (PT)
141470: José Guimarães (PT)
141488: Paulo Teixeira (PT)
141516: Paulo Abi-Ackel (PSDB)
141518: Paulinho da Força (SOLIDARIEDADE)
141531: Rodrigo de Castro (UNIÃO)
141555: Vinicius Carvalho (REPUBLICANOS)
143084: Tiago Dimas (PODE)
143632: Domingos Neto (PSD)
150418: Márcio Marinho (REPUBLICANOS)
152605: Glauber Braga (PSOL)
154178: Milton Vieira (REPUBLICANOS)
156190: Marcel van Hattem (NOVO)
157130: Túlio Gadêlha (REDE)
160508: Afonso Florence (PT)
160510: Luis Tibé (AVANTE)
160512: Aureo Ribeiro (SOLIDARIEDADE)
160518: Weliton Prado (SOLIDARIEDADE)
160527: Aguinaldo Ribeiro (PP)
160531: Jhonatan de Jesus (REPUBLICANOS)
160534: Lauriete (PR)
160535: Marcon (PT)
160538: Bohn Gass (PT)
160545: Alexandre Leite (UNIÃO)
160553: Antonio Brito (PSD)
160556: Padre João (PT)
160558: Paulo Freire Costa (PL)
160559: Alceu Moreira (MDB)
160569: Waldenor Pereira (PT)
160575: Erika Kokay (PT)
160588: Diego Andrade (PSD)
160598: Flávia Morais (PDT)
160599: Dimas Fabiano (PP)
160600: Arthur Oliveira Maia (UNIÃO)
160601: Pr. Marco Feliciano (PL)
160604: Pedro Uczai (PT)
160610: Valmir Assunção (PT)
160632: Zé Silva (SOLIDARIEDADE)
160635: Ruy Carneiro (PODE)
160640: Eros Biondini (PL)
160641: Professora Marcivania (PCdoB)
160642: Pastor Eurico (PL)
160665: Augusto Coutinho (REPUBLICANOS)
160666: Félix Mendonça Júnior (PDT)
160673: Giovani Cherini (PL)
160674: Hugo Motta (REPUBLICANOS)
160758: Domingos Sávio (PL)
160976: Tiririca (PSD)
161550: Emidinho Madeira (PL)
162332: Luiz Nishimori (PSD)
164360: Carmen Zanotto (CIDADANIA)
166402: Magda Mofatto (PRD)
171617: Paulão (PT)
171623: Fabio Reis (PSD)
178829: Capitão Augusto (PL)
178831: ANDRÉ ABDON (PP)
178832: Leandre (PSD)
178835: Afonso Motta (PDT)
178843: Marx Beltrão (PP)
178854: Elmar Nascimento (UNIÃO)
178857: Jorge Solla (PT)
178858: Mário Negromonte Jr. (PP)
178860: Paulo Azi (UNIÃO)
178866: Luizianne Lins (PT)
178871: Evair Vieira de Melo (PP)
178873: Helder Salomão (PT)
178876: Célio Silveira (MDB)
178881: Aluisio Mendes (REPUBLICANOS)
178882: André Fufuca (PP)
178884: Hildo Rocha (MDB)
178886: Juscelino Filho (UNIÃO)
178887: Rubens Pereira Júnior (PT)
178895: Misael Varella (PSD)
178896: Newton Cardoso Jr (MDB)
178908: Delegado Éder Mauro (PL)
178909: Hélio Leite (UNIÃO)
178910: Joaquim Passarinho (PL)
178927: Aliel Machado (PV)
178928: Christiane de Souza Yared (PR)
178929: Diego Garcia (REPUBLICANOS)
178931: Luciano Ducci (PSB)
178933: Sergio Souza (MDB)
178934: Toninho Wandscheer (PP)
178945: Rosangela Gomes (REPUBLICANOS)
178946: Soraya Santos (PL)
178947: Sóstenes Cavalcante (PL)
178954: Lucio Mosquini (MDB)
178962: Carlos Gomes (REPUBLICANOS)
178963: Covatti Filho (PP)
178964: Heitor Schuch (PSB)
178966: Geovania de Sá (PSDB)
178970: João Daniel (PT)
178972: Alex Manente (CIDADANIA)
178975: Baleia Rossi (MDB)
178983: Marcio Alvino (PL)
178985: Miguel Lombardi (PL)
178986: Nilto Tatto (PT)
178987: Orlando Silva (PCdoB)
178990: Samuel Moreira (PSDB)
178992: Vitor Lippi (PSDB)
178993: Carlos Henrique Gaguim (UNIÃO)
178997: Moses Rodrigues (UNIÃO)
179001: Márcio Biolchi (MDB)
188097: Merlong Solano (PT)
191923: Flávio Nogueira (PT)
198197: Greyce Elias (AVANTE)
198783: Emanuel Pinheiro Neto (MDB)
204351: Dr. Luiz Ovando (PP)
204353: Aline Gurgel (REPUBLICANOS)
204355: Da Vitoria (PP)
204356: Amaro Neto (REPUBLICANOS)
204358: Beto Pereira (PSDB)
204360: Silvia Cristina (PP)
204361: Carlos Chiodini (MDB)
204362: Ricardo Guidi (PL)
204364: Eli Borges (PL)
204365: Gilson Marques (NOVO)
204366: Rodrigo Coelho (PSB)
204367: Daniel Freitas (PL)
204368: Fabio Schiochet (UNIÃO)
204369: Caroline de Toni (PL)
204372: Julio Cesar Ribeiro (REPUBLICANOS)
204374: Bia Kicis (PL)
204376: Coronel Armando (PP)
204378: Coronel Chrisóstomo (PL)
204379: Acácio Favacho (MDB)
204386: Jose Mario Schreiner (DEM)
204387: Sargento Fahur (PSD)
204388: Bibo Nunes (PL)
204390: Professor Alcides (PL)
204391: José Nelto (UNIÃO)
204394: Gervásio Maia (PSB)
204395: Pedro Lupion (REPUBLICANOS)
204396: Vermelho (PP)
204398: Felipe Francischini (UNIÃO)
204400: Aline Sleutjes (PSL)
204404: Lucas Redecker (PSDB)
204406: Pedro Westphalen (PP)
204407: Fernanda Melchionna (PSOL)
204408: Gustinho Ribeiro (REPUBLICANOS)
204409: Daniel Trzeciak (PSDB)
204410: Luisa Canziani (PSD)
204411: Filipe Barros (PL)
204412: Dr. Zacharias Calil (UNIÃO)
204415: Nereu Crispim (PSL)
204416: Sanderson (PL)
204419: Glaustin da Fokus (PODE)
204422: Ossesio Silva (REPUBLICANOS)
204423: André Ferreira (PL)
204425: Silvio Costa Filho (REPUBLICANOS)
204426: Carlos Veras (PT)
204427: Fernando Rodolfo (PL)
204431: Marcos Aurélio Sampaio (PSD)
204432: Marina Santos (SOLIDARIEDADE)
204434: Dra. Vanda Milani (SOLIDARIEDADE)
204436: Isnaldo Bulhões Jr. (MDB)
204441: Otoni de Paula (MDB)
204444: Helio Lopes (PL)
204448: Luiz Antonio Corrêa (PP)
204449: Nelson Barbudo (PL)
204450: Doutor Luizinho (PP)
204453: Natália Bonavides (PT)
204455: Luiz Lima (NOVO)
204456: Gutemberg Reis (MDB)
204457: Juninho do Pneu (UNIÃO)
204459: Daniela do Waguinho (UNIÃO)
204460: Carlos Jordy (PL)
204461: Paulo Ganime (NOVO)
204462: Chris Tonietto (PL)
204464: Talíria Petrone (PSOL)
204472: José Medeiros (PL)
204473: General Girão (PL)
204474: Juarez Costa (MDB)
204476: Chiquinho Brazão (S.PART.)
204477: Felício Laterça (PSL)
204479: Nicoletti (UNIÃO)
204480: Rogério Correia (PT)
204481: Dr. Jaziel (PL)
204482: Euclydes Pettersen (REPUBLICANOS)
204484: General Peternelli (PSL)
204485: Luiz Carlos Motta (PL)
204488: Célio Studart (PSD)
204489: Robério Monteiro (PDT)
204491: Gilberto Abramo (REPUBLICANOS)
204492: Paulo Guedes (PT)
204494: Fred Costa (PRD)
204495: Airton Faleiro (PT)
204496: Celso Sabino (UNIÃO)
204497: Júnior Ferrari (PSD)
204498: Olival Marques (MDB)
204501: Alencar Santana (PT)
204503: Alexandre Padilha (PT)
204504: Cezinha de Madureira (PSD)
204506: Marcos Pereira (REPUBLICANOS)
204507: Carla Zambelli (PL)
204508: Igor Timo (PSD)
204511: David Soares (UNIÃO)
204512: Delegado Marcelo Freitas (UNIÃO)
204513: Guiga Peixoto (PSL)
204514: Coronel Tadeu (PL)
204515: André Janones (AVANTE)
204518: Dr. Frederico (PRD)
204519: Tiago Mitraud (NOVO)
204520: Junio Amaral (PL)
204522: Marco Bertaiolli (PSD)
204524: Pinheirinho (PP)
204525: Rosana Valle (PL)
204526: Luiz Philippe de Orleans e Bragança (PL)
204528: Adriana Ventura (NOVO)
204531: Guilherme Derrite (PP)
204533: Idilvan Alencar (PDT)
204534: Tabata Amaral (PSB)
204535: Sâmia Bomfim (PSOL)
204536: Kim Kataguiri (UNIÃO)
204539: Hercílio Coelho Diniz (MDB)
204540: Maria Rosas (REPUBLICANOS)
204541: Eduardo Bismarck (PDT)
204549: AJ Albuquerque (PP)
204550: Júnior Mano (PSB)
204553: Pastor Sargento Isidório (AVANTE)
204559: Zé Neto (PT)
204566: Marreca Filho (PRD)
204567: Raimundo Costa (PODE)
204570: Pastor Gil (PL)
204571: Alex Santana (REPUBLICANOS)
204572: Capitão Alberto Neto (PL)
204573: Otto Alencar Filho (PSD)
205476: Charles Fernandes (PSD)
205550: Jorge Braz (REPUBLICANOS)
209189: Joseildo Ramos (PT)
213274: Ricardo Silva (PSD)
213762: Carla Dickson (UNIÃO)
215043: Josivaldo JP (PSD)
62881: Danilo Forte (UNIÃO)
66828: Fausto Pinato (PP)
68720: Fábio Henrique (UNIÃO)
69871: Bacelar (PV)
72442: Felipe Carreras (PSB)
73433: Arlindo Chinaglia (PT)
73441: Celso Russomanno (REPUBLICANOS)
73466: Rubens Bueno (PPS)
73486: Pompeo de Mattos (PDT)
73531: Ivan Valente (PSOL)
73604: Rui Falcão (PT)
73692: Osmar Terra (PL)
73701: Benedita da Silva (PT)
73788: Ricardo Barros (PP)
73801: Renildo Calheiros (PCdoB)
73808: Sérgio Brito (PSD)
74043: Wellington Roberto (PL)
74044: Wilson Santiago (REPUBLICANOS)
74052: Bosco Costa (PL)
74057: Alice Portugal (PCdoB)
74060: Daniel Almeida (PCdoB)
74075: Elcione Barbalho (MDB)
74079: José Priante (MDB)
74090: Átila Lins (PSD)
74156: Leonardo Monteiro (PT)
74158: Mário Heringer (PDT)
74159: Odair Cunha (PT)
74160: Patrus Ananias (PT)
74161: Reginaldo Lopes (PT)
74262: Carlos Sampaio (PSD)
74273: Jefferson Campos (PL)
74283: Vicentinho (PT)
74299: Leônidas Cristino (PDT)
74317: Júlio Cesar (PSD)
74356: Silas Câmara (REPUBLICANOS)
74371: Rubens Otoni (PT)
74376: Vander Loubet (PT)
74398: Maria do Rosário (PT)
74400: Paulo Pimenta (PT)
74459: Átila Lira (PSB)
74467: Damião Feliciano (UNIÃO)
74537: Claudio Cajado (PP)
74554: José Rocha (UNIÃO)
74574: Paulo Magalhães (PSD)
74585: Lincoln Portela (PL)
74646: Aécio Neves (PSDB)
74784: Luiza Erundina (PSOL)
74848: Jandira Feghali (PCdoB)
75431: Maurício Dziedricki (PTB)
81055: Márcio Jerry (PCdoB)
81297: Dra. Soraya Manato (PSL)
92102: Leur Lomanto Júnior (UNIÃO)
92346: Eduardo Bolsonaro (PL)
92699: Fernando Monteiro (PSD)
98057: Lafayette de Andrada (REPUBLICANOS)

Quantidade de deputados por partido na coalizão:

AVANTE: 4
CIDADANIA: 3
DEM: 1
MDB: 20
NOVO: 6
PCdoB: 7
PDT: 10
PL: 49
PODE: 4
PP: 28
PPS: 1
PR: 2
PRD: 4
PSB: 9
PSD: 29
PSDB: 9
PSL: 6
PSOL: 6
PT: 44
PTB: 1
PV: 2
REDE: 1
REPUBLICANOS: 27
S.PART.: 1
SOLIDARIEDADE: 6
UNIÃO: 28'''

''' Deputados selecionados:

107283: Gleisi Hoffmann (PT)
109429: Benes Leocádio (UNIÃO)
116379: Darci de Matos (PSD)
121948: Adriano do Baldy (PP)
122158: Pedro Paulo (PSD)
122974: Pedro Lucas Fernandes (UNIÃO)
132504: Enio Verri (PT)
133439: André Figueiredo (PDT)
133810: Marcelo Moraes (PL)
136811: Afonso Hamm (PP)
137070: Vicentinho Júnior (PP)
139285: Lídice da Mata (PSB)
141391: Arnaldo Jardim (CIDADANIA)
141398: Carlos Zarattini (PT)
141408: Cleber Verde (MDB)
141411: Dagoberto Nogueira (PSDB)
141421: Eduardo da Fonte (PP)
141450: Hugo Leal (PSD)
141458: João Carlos Bacelar (PL)
141459: João Maia (PP)
141464: José Airton Félix Cirilo (PT)
141470: José Guimarães (PT)
141488: Paulo Teixeira (PT)
141518: Paulinho da Força (SOLIDARIEDADE)
141531: Rodrigo de Castro (UNIÃO)
141555: Vinicius Carvalho (REPUBLICANOS)
143084: Tiago Dimas (PODE)
143632: Domingos Neto (PSD)
150418: Márcio Marinho (REPUBLICANOS)
152605: Glauber Braga (PSOL)
154178: Milton Vieira (REPUBLICANOS)
156190: Marcel van Hattem (NOVO)
157130: Túlio Gadêlha (REDE)
160508: Afonso Florence (PT)
160510: Luis Tibé (AVANTE)
160512: Aureo Ribeiro (SOLIDARIEDADE)
160527: Aguinaldo Ribeiro (PP)
160531: Jhonatan de Jesus (REPUBLICANOS)
160535: Marcon (PT)
160538: Bohn Gass (PT)
160553: Antonio Brito (PSD)
160556: Padre João (PT)
160558: Paulo Freire Costa (PL)
160559: Alceu Moreira (MDB)
160569: Waldenor Pereira (PT)
160575: Erika Kokay (PT)
160598: Flávia Morais (PDT)
160599: Dimas Fabiano (PP)
160600: Arthur Oliveira Maia (UNIÃO)
160601: Pr. Marco Feliciano (PL)
160604: Pedro Uczai (PT)
160610: Valmir Assunção (PT)
160632: Zé Silva (SOLIDARIEDADE)
160635: Ruy Carneiro (PODE)
160640: Eros Biondini (PL)
160641: Professora Marcivania (PCdoB)
160642: Pastor Eurico (PL)
160665: Augusto Coutinho (REPUBLICANOS)
160673: Giovani Cherini (PL)
160674: Hugo Motta (REPUBLICANOS)
160758: Domingos Sávio (PL)
161550: Emidinho Madeira (PL)
162332: Luiz Nishimori (PSD)
164360: Carmen Zanotto (CIDADANIA)
171617: Paulão (PT)
171623: Fabio Reis (PSD)
178829: Capitão Augusto (PL)
178831: ANDRÉ ABDON (PP)
178832: Leandre (PSD)
178835: Afonso Motta (PDT)
178843: Marx Beltrão (PP)
178857: Jorge Solla (PT)
178858: Mário Negromonte Jr. (PP)
178860: Paulo Azi (UNIÃO)
178866: Luizianne Lins (PT)
178871: Evair Vieira de Melo (PP)
178873: Helder Salomão (PT)
178876: Célio Silveira (MDB)
178881: Aluisio Mendes (REPUBLICANOS)
178884: Hildo Rocha (MDB)
178886: Juscelino Filho (UNIÃO)
178887: Rubens Pereira Júnior (PT)
178895: Misael Varella (PSD)
178896: Newton Cardoso Jr (MDB)
178909: Hélio Leite (UNIÃO)
178910: Joaquim Passarinho (PL)
178929: Diego Garcia (REPUBLICANOS)
178931: Luciano Ducci (PSB)
178933: Sergio Souza (MDB)
178934: Toninho Wandscheer (PP)
178945: Rosangela Gomes (REPUBLICANOS)
178947: Sóstenes Cavalcante (PL)
178954: Lucio Mosquini (MDB)
178962: Carlos Gomes (REPUBLICANOS)
178963: Covatti Filho (PP)
178964: Heitor Schuch (PSB)
178966: Geovania de Sá (PSDB)
178970: João Daniel (PT)
178972: Alex Manente (CIDADANIA)
178975: Baleia Rossi (MDB)
178983: Marcio Alvino (PL)
178985: Miguel Lombardi (PL)
178986: Nilto Tatto (PT)
178987: Orlando Silva (PCdoB)
178992: Vitor Lippi (PSDB)
178993: Carlos Henrique Gaguim (UNIÃO)
178997: Moses Rodrigues (UNIÃO)
179001: Márcio Biolchi (MDB)
188097: Merlong Solano (PT)
191923: Flávio Nogueira (PT)
198197: Greyce Elias (AVANTE)
198783: Emanuel Pinheiro Neto (MDB)
204351: Dr. Luiz Ovando (PP)
204353: Aline Gurgel (REPUBLICANOS)
204355: Da Vitoria (PP)
204356: Amaro Neto (REPUBLICANOS)
204360: Silvia Cristina (PP)
204361: Carlos Chiodini (MDB)
204362: Ricardo Guidi (PL)
204364: Eli Borges (PL)
204367: Daniel Freitas (PL)
204368: Fabio Schiochet (UNIÃO)
204369: Caroline de Toni (PL)
204372: Julio Cesar Ribeiro (REPUBLICANOS)
204374: Bia Kicis (PL)
204376: Coronel Armando (PP)
204378: Coronel Chrisóstomo (PL)
204387: Sargento Fahur (PSD)
204388: Bibo Nunes (PL)
204390: Professor Alcides (PL)
204391: José Nelto (UNIÃO)
204394: Gervásio Maia (PSB)
204395: Pedro Lupion (REPUBLICANOS)
204396: Vermelho (PP)
204398: Felipe Francischini (UNIÃO)
204404: Lucas Redecker (PSDB)
204406: Pedro Westphalen (PP)
204407: Fernanda Melchionna (PSOL)
204408: Gustinho Ribeiro (REPUBLICANOS)
204409: Daniel Trzeciak (PSDB)
204410: Luisa Canziani (PSD)
204411: Filipe Barros (PL)
204412: Dr. Zacharias Calil (UNIÃO)
204416: Sanderson (PL)
204419: Glaustin da Fokus (PODE)
204422: Ossesio Silva (REPUBLICANOS)
204423: André Ferreira (PL)
204425: Silvio Costa Filho (REPUBLICANOS)
204426: Carlos Veras (PT)
204427: Fernando Rodolfo (PL)
204436: Isnaldo Bulhões Jr. (MDB)
204441: Otoni de Paula (MDB)
204444: Helio Lopes (PL)
204449: Nelson Barbudo (PL)
204450: Doutor Luizinho (PP)
204453: Natália Bonavides (PT)
204455: Luiz Lima (NOVO)
204456: Gutemberg Reis (MDB)
204459: Daniela do Waguinho (UNIÃO)
204460: Carlos Jordy (PL)
204462: Chris Tonietto (PL)
204464: Talíria Petrone (PSOL)
204472: José Medeiros (PL)
204473: General Girão (PL)
204474: Juarez Costa (MDB)
204476: Chiquinho Brazão (S.PART.)
204479: Nicoletti (UNIÃO)
204480: Rogério Correia (PT)
204481: Dr. Jaziel (PL)
204482: Euclydes Pettersen (REPUBLICANOS)
204485: Luiz Carlos Motta (PL)
204488: Célio Studart (PSD)
204491: Gilberto Abramo (REPUBLICANOS)
204492: Paulo Guedes (PT)
204494: Fred Costa (PRD)
204495: Airton Faleiro (PT)
204496: Celso Sabino (UNIÃO)
204497: Júnior Ferrari (PSD)
204501: Alencar Santana (PT)
204503: Alexandre Padilha (PT)
204504: Cezinha de Madureira (PSD)
204506: Marcos Pereira (REPUBLICANOS)
204507: Carla Zambelli (PL)
204511: David Soares (UNIÃO)
204512: Delegado Marcelo Freitas (UNIÃO)
204514: Coronel Tadeu (PL)
204518: Dr. Frederico (PRD)
204520: Junio Amaral (PL)
204522: Marco Bertaiolli (PSD)
204524: Pinheirinho (PP)
204525: Rosana Valle (PL)
204526: Luiz Philippe de Orleans e Bragança (PL)
204528: Adriana Ventura (NOVO)
204531: Guilherme Derrite (PP)
204533: Idilvan Alencar (PDT)
204534: Tabata Amaral (PSB)
204535: Sâmia Bomfim (PSOL)
204536: Kim Kataguiri (UNIÃO)
204540: Maria Rosas (REPUBLICANOS)
204541: Eduardo Bismarck (PDT)
204549: AJ Albuquerque (PP)
204550: Júnior Mano (PSB)
204553: Pastor Sargento Isidório (AVANTE)
204559: Zé Neto (PT)
204570: Pastor Gil (PL)
204572: Capitão Alberto Neto (PL)
204573: Otto Alencar Filho (PSD)
205476: Charles Fernandes (PSD)
205550: Jorge Braz (REPUBLICANOS)
209189: Joseildo Ramos (PT)
213274: Ricardo Silva (PSD)
213762: Carla Dickson (UNIÃO)
62881: Danilo Forte (UNIÃO)
66828: Fausto Pinato (PP)
68720: Fábio Henrique (UNIÃO)
69871: Bacelar (PV)
72442: Felipe Carreras (PSB)
73433: Arlindo Chinaglia (PT)
73486: Pompeo de Mattos (PDT)
73531: Ivan Valente (PSOL)
73604: Rui Falcão (PT)
73692: Osmar Terra (PL)
73701: Benedita da Silva (PT)
73788: Ricardo Barros (PP)
73801: Renildo Calheiros (PCdoB)
73808: Sérgio Brito (PSD)
74043: Wellington Roberto (PL)
74044: Wilson Santiago (REPUBLICANOS)
74052: Bosco Costa (PL)
74057: Alice Portugal (PCdoB)
74060: Daniel Almeida (PCdoB)
74075: Elcione Barbalho (MDB)
74090: Átila Lins (PSD)
74156: Leonardo Monteiro (PT)
74158: Mário Heringer (PDT)
74159: Odair Cunha (PT)
74160: Patrus Ananias (PT)
74161: Reginaldo Lopes (PT)
74262: Carlos Sampaio (PSD)
74273: Jefferson Campos (PL)
74283: Vicentinho (PT)
74299: Leônidas Cristino (PDT)
74317: Júlio Cesar (PSD)
74356: Silas Câmara (REPUBLICANOS)
74371: Rubens Otoni (PT)
74376: Vander Loubet (PT)
74398: Maria do Rosário (PT)
74400: Paulo Pimenta (PT)
74467: Damião Feliciano (UNIÃO)
74537: Claudio Cajado (PP)
74585: Lincoln Portela (PL)
74784: Luiza Erundina (PSOL)
74848: Jandira Feghali (PCdoB)
92102: Leur Lomanto Júnior (UNIÃO)
92346: Eduardo Bolsonaro (PL)
92699: Fernando Monteiro (PSD)
98057: Lafayette de Andrada (REPUBLICANOS)

Quantidade de deputados por partido na coalizão:

AVANTE: 3
CIDADANIA: 3
MDB: 16
NOVO: 3
PCdoB: 6
PDT: 8
PL: 47
PODE: 3
PP: 26
PRD: 2
PSB: 7
PSD: 23
PSDB: 5
PSOL: 6
PT: 44
PV: 1
REDE: 1
REPUBLICANOS: 25
S.PART.: 1
SOLIDARIEDADE: 3
UNIÃO: 24'''